src/diffusers/models/unet_3d_blocks.py

```diff
@@ -15,8 +15,9 @@
 import torch
 from torch import nn
 
-from .resnet import Downsample2D, ResnetBlock2D, TemporalConvLayer, Upsample2D
+from .resnet import Downsample2D, Downsample3D, ResnetBlock2D, ResnetBlock3D, TemporalConvLayer, Upsample2D, Upsample3D
 from .transformer_2d import Transformer2DModel
```

Comment:
Could we try to just create new blocks here instead of modifying the existing ones? Think that would be easier :-)

Reply:
@patrickvonplaten, can you clarify here please ? `Downsample3D`, `ResnetBlock3D`, `Upsample3D` are new blocks.